server/src/main/java/org/opensearch/cluster/block/ClusterBlocks.java

```diff
@@ -204,6 +204,13 @@ public ClusterBlockException globalBlockedException(ClusterBlockLevel level) {
         return new ClusterBlockException(global(level));
     }
 
+    public IndexCreateBlockException createIndexBlockedException(ClusterBlockLevel level) {
+        if (global(level).isEmpty()) {
```

Comment:
Can we use existing methods like globalBlocked or hasGlobalBlockWithLevel?

Reply:
Addressed the comments.